apps/apple/FluentUITester/macos/FluentUITester-macOS/Config/Common.xcconfig

```diff
@@ -23,8 +24,8 @@ GCC_WARN_UNUSED_FUNCTION = YES
 GCC_WARN_UNUSED_VARIABLE = YES
 SWIFT_TREAT_WARNINGS_AS_ERRORS = YES
 
-INFOPLIST_FILE = FluentUIReactNativeTester_macOS/Info.plist
+INFOPLIST_FILE = FluentUITester-macos/Info.plist
 
-// Versions
-MACOSX_DEPLOYMENT_TARGET = 10.13
+SDKROOT = macosx;
+MACOSX_DEPLOYMENT_TARGET = 10.14;
```

Comment:
why this change? I thought this was minOS, which is still 10.13?

Reply:
Didn't mean to change it- 10.13 is correct